src/provider/connectors.ts

```diff
@@ -50,8 +50,12 @@ export const isChainIdSupported = (chainId: number): boolean => {
 
 
 const backupUrls = {};
-backupUrls[supportedChainId.toString()] = RPC_URLS[supportedChainId];
-
+backupUrls[supportedChainId] = RPC_URLS[supportedChainId];
+console.log({
```

Comment:
Do you want to keep this log statement in here for now?

Reply:
removed